pkg/workceptor/kubernetes.go

```diff
@@ -203,7 +206,15 @@ func (kw *kubeUnit) createPod(env map[string]string) error {
 			return kw.clientset.CoreV1().Pods(ked.KubeNamespace).Watch(kw.ctx, options)
 		},
 	}
-	ev, err := watch2.UntilWithSync(kw.ctx, lw, &corev1.Pod{}, nil, podRunningAndReady)
+
+	timeout, err := strconv.Atoi(ked.PodPendingTimeout)
```

Comment:
We should be using time.ParseDuration instead https://golang.org/pkg/time/#ParseDuration

it allow user the specify units of time to, e.g. `1.5h`, `20m`, etc

Reply:
I have made the required changes.